homeassistant/components/homekit_controller/config_flow.py

```diff
@@ -237,12 +237,9 @@ def __init__(self):
 
         return self.async_show_form(
             step_id='pair',
-            description_placeholders={
-                'model': self.model,
-            },
             errors=errors,
             data_schema=vol.Schema({
-                vol.Required('pairing_code'):  vol.All(str, vol.Strip),
```

Comment:
`vol.Strip` should be supported I think.

https://github.com/balloob/voluptuous-serialize/pull/7

Reply:
I got a serialize error traceback about it, and removing this change fixed it. Will re-revert it and post it.

Reply:
Ah, apparently it's been 2 months already since I started working on this and i just needed to pip install!